README.md

```diff
@@ -29,22 +31,54 @@ The configuration file `(src/main/resources/config.json)` specifies one or more
 }
 ```
 
-### Usage
+Table names will be of the format [tableNamePrefix]YYYYMMDD for use in table date range queries.
+
+## Job Inserter Configuration
+
+To add a query, you need to modify the configuration file `(src/main/resources/job-inserter-config.json)`
+and add a query under `src/main/resources/queries/`
+
+The configuration file follows the following schema:
+
+```json
+{
+  "projectId": "destination-project",
+  "configuration":
+  {
+    ... 
+  },
+  "every4Hours": [
+    {
+      "destinationTableId": "destination_table_name",
+      "fileName": "queries/query-file-name.sql"
+    }
+  ],
+  "every24Hours": [
+    ...
+  ]
+}
+```
+
+The query result will be stored in `[client-side-events:Aggregate_Tables.destination_table_name]`.
```

Comment:
@tejohnso the project name is configurable, but in the interest of simplicity, I made it a "global" field that is configured once for all queries.

It wouldn't be too hard to allow a user to set the project and dataset per query, but that would be an extra 2 fields of configuration for each request.